src/crucible.rs

```diff
@@ -41,7 +41,7 @@ impl Default for Crucible {
             plan_runner: None,
             camera_jump: None,
             frozen_fabric: None,
-            iterations_per_frame: 100,
+            iterations_per_frame: 5,
```

Comment:
Do we ever want to make this easier to change on-the-fly? Slow and fast simulation are both very useful but it's a pain to switch